### PR TITLE
Ym/imp smoke test and setup

### PR DESCRIPTION
**Pull request by yefim-mironyuk**
Logger.set_logger(Logger()) removed from _assertions.py_ cuz we dont have Logging in assertions anymore
_config.py_ file changed. Variables are in Classes now
creds should be entered from command line to run tests
setup for tests implemented
new methods in _browser_helper.py_ added
log_in smoke test implemented
Locators naming changed in _main_page_locators.py_
New locators added in _main_page_locators.py_
Loguru in pipfile added

### CODE REVIEW

**Review by kit-91-daniil — APPROVED** [review]

File: config.py [hunk 1]
```diff
@@ -1,2 +1,6 @@
-wait_time_s = 5
-MAIN_PAGE_URL = "https://www.ryanair.com/gb/en"
+class BrowserHelperSettings:
+    WAIT_TIME_SECS = 5
+
```
Comment (relaxing-observer) [inline config.py:3]: Consider using default cred here as 
class DefaultCreds:
    USERNAME = ""
    PASSWORD = ""

File: tests/conftest.py [hunk 2]
```diff
@@ -73,3 +86,20 @@ def main_page_car_hire_tab(browser):
 @pytest.fixture(scope="module")
 def main_page_search_hotels_tab(browser):
     return MainPageSearchHotelsTab(browser=browser, url=browser.current_url)
+
+
+@pytest.fixture(scope="module")
+def assertions(browser):
+    return Assertions(browser=browser)
+
```
Comment (relaxing-observer) [inline tests/conftest.py:92]: I think we should initialize it as self.assertions in BasePage such as ElementInteractions

**Review by kit-91-daniil — APPROVED** [review]
I like it)

File: tests/conftest.py [hunk 3]
```diff
@@ -73,3 +86,20 @@ def main_page_car_hire_tab(browser):
...
+
+@pytest.fixture()
+def user(request):
+    user_cmd = request.config.getoption("user")
+    return user_cmd
+
```
Comment (relaxing-observer) [inline tests/conftest.py:100]: if user_cmd == "None":
        return config.DefaultCreds.USERNAME
    else:
        return user_cmd
I consider add this

File: tests/conftest.py [hunk 4]
```diff
@@ -73,3 +86,20 @@ def main_page_car_hire_tab(browser):
...
+
+@pytest.fixture()
+def password(request):
+    password_cmd = request.config.getoption("password")
+    return password_cmd
```
Comment (relaxing-observer) [inline tests/conftest.py:107]: if password_cmd== "None":
        return config.DefaultCreds.PASSWORD
    else:
        return password_cmd
I consider add this

**Review by relaxing-observer — CHANGES_REQUESTED** [review]

**Review by relaxing-observer — APPROVED** [review]

File: config.py (hunk 1 above)
Comment (yefim-mironyuk) [reply config.py:3]: Done

File: tests/conftest.py (hunk 2 above)
Comment (yefim-mironyuk) [reply tests/conftest.py:92]: And write something like *page_name*.assertions.*method_name* when we can just assertions.*method_name*

File: tests/conftest.py (hunk 4 above)
Comment (yefim-mironyuk) [reply tests/conftest.py:107]: Done

File: tests/conftest.py (hunk 3 above)
Comment (yefim-mironyuk) [reply tests/conftest.py:100]: Done

File: tests/conftest.py (hunk 2 above)
Comment (relaxing-observer) [reply tests/conftest.py:92]: up to u

**Review by TA-user — APPROVED** [review]

**Pull request merged**